spec/factories.rb

```diff
@@ -6,6 +7,29 @@
     sequence(:name) { |n| "Name #{n}" }
     sequence(:started_at) { |n| n.hours.from_now }
     owner
+
+    trait :intrepid do
+      name 'Intrepid Labs'
+      lat 42.36708
+      lon (-71.08035)
+    end
+
+    trait :museum do
+      name 'Museum of Science'
+      lat 42.36673
+      lon (-71.06761)
+    end
+
+    trait :cambridge do
+      name 'Central Square'
+      lat 42.36562
+      lon (-71.10385)
+    end
+
+    trait :chicago do
```

Comment:
weird -- I added one, saved and committed, but it's not showing up here. It's the Jackson Street Bridge.